driving_log_replayer/scripts/localization_evaluator_node.py

```diff
@@ -144,7 +149,7 @@ def add_reliability_frame(
                 }
             ],
         }
-        self.__reliability_msg = f"{self.__reliability_condition['Method']} Sequential NG Count: {self.__reliability_ng_seq} (Total Test: {self.__reliability_total})"
+        self.__reliability_msg = f"{self.__reliability_condition['Method']} Sequential NG Count: {self.__reliability_ng_seq} (Total Test: {self.__reliability_total}, Average: {statistics.mean(self.__reliability_list)}, StdDev: {statistics.pstdev(self.__reliability_list)})"
```

Comment:
```suggestion
        self.__reliability_msg = f"{self.__reliability_condition['Method']} Sequential NG Count: {self.__reliability_ng_seq} (Total Test: {self.__reliability_total}, Average: {statistics.mean(self.__reliability_list):.5f}, StdDev: {statistics.pstdev(self.__reliability_list):.5f})"
```
How about this to limit the number of decimals to be logged?